src/components/GenQRcode.vue

```diff
@@ -24,14 +29,13 @@ export default {
       this.qrcode,
       {
         color: {
-          dark: this.color[0], 
-          light: this.color[1],
+          dark: this.dark,
+          light: this.light,
         },
         version: this.size,
       },
-      function(error) {
-        if (error) alert(error)
-        alert('success!')
+      err => {
+        if (err != null) _procError(err)
```

Comment:
no need to check for  : if (err != null)
coz err 100% not null, coz it's in the catch clause